contracts/p1/RToken.sol

```diff
@@ -79,8 +80,8 @@ contract RTokenP1 is ComponentP1, ERC20PermitUpgradeable, IRToken {
         furnace = main_.furnace();
 
         mandate = mandate_;
-        setIssuanceThrottleParams(issuanceThrottleParams_);
         setRedemptionThrottleParams(redemptionThrottleParams_);
+        setIssuanceThrottleParams(issuanceThrottleParams_);
```

Comment:
calling the hybrid setter here seems slightly better. would have to make `public`